src/herder/test/HerderTests.cpp

```diff
@@ -471,7 +473,9 @@ testTxSetWithFeeBumps(uint32 protocolVersion)
                 feeBump(*app, account2, tx2, minBalance2 - minBalance0 - 199);
             txSet->add(fb2);
 
-            checkTrimCheck({fb1, fb2});
+            std::vector<TransactionFrameBasePtr> txs{fb1, fb2};
+            std::sort(txs.begin(), txs.end());
```

Comment:
I don't think this is ideal. While I agree it solves the problem, it leaves us open to having this problem again if we ever modify these tests. You should make this change in `checkTrimCheck` so that anyone who uses this function will have automatically accounted for it.

Reply:
Also it would probably be better if you sorted by hash-order rather than pointer-order (both here and on line 459) since the pointers passed in to `checkTrimCheck` aren't guaranteed to be the same ones that are actually in the `TransactionSet` (although in practice I think this is true today).

Reply:
Agree with moving this into `checkTrimCheck` - fixed

re: hash order. This doesn't make sense here: tests check that those vectors (of shared pointers) are equal, and performs pointer comparison, we don't have a viable comparison function for `TransactionFrameBase`.

Reply:
Oh good point about the hash order. Someone definitely could misuse this function then, but if that does happen then I'm fairly certain that they would find out deterministically so it isn't too bad. Fixing this would be more effort than it's worth.